src/styles/pages/Login.styles.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+
+export const Header = styled.h1`
+  text-align: center;
+`;
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 450px;
+  height: 240px;
+  margin-top: 128px;
+  margin-left: 480px;
+  background-color: rgb(245, 245, 245);
+  padding: 70px 60px;
+`;
```

Comment:
I reviewed your code and I saw that your login component is not centered, look, I show you a solution to center it, it is only necessary to add one more style and modify a current one as seen in the following code.

```suggestion
export const ContainerWrapper = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center
`;

export const Container = styled.div`
  width: 450px;
  height: 300px;
  margin-top: 128px;
  background-color: rgb(245, 245, 245);
  padding: 70px 60px;
`;
```